src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -1749,13 +1749,15 @@ static inline int MPIDI_OFI_init_hints(struct fi_info *hints)
 #endif
     }
     hints->tx_attr->op_flags = FI_COMPLETION;
+    hints->tx_attr->msg_order = FI_ORDER_SAS;
     /* direct RMA operations supported only with delivery complete mode,
      * else (AM mode) delivery complete is not required */
-    if (MPIDI_OFI_ENABLE_RMA)
+    if (MPIDI_OFI_ENABLE_RMA) {
         hints->tx_attr->op_flags |= FI_DELIVERY_COMPLETE;
-    /* Apply most restricted msg order in hints for RMA. */
-    hints->tx_attr->msg_order =
-        FI_ORDER_SAS | FI_ORDER_RAR | FI_ORDER_RAW | FI_ORDER_WAR | FI_ORDER_WAW;
+        /* Apply most restricted msg order in hints for RMA. */
```

Comment:
Could you please change the comment to : `Apply most restricted msg order in hints for ATOMIC RMA ` ?

Reply:
Sure.

@minsii @hajimefu Should these two commits squashed into one?

Reply:
Well, I thought this was another fix to previous code which might be better to separate.  But it is OK to squash in my opinion.

Reply:
This comment has been applied